addon/components/ember-tooltip-base.js

```diff
@@ -160,6 +160,14 @@ export default Component.extend({
     return `${this.get('elementId')}-wormhole`;
   }),
 
+  wormholeEl: computed('wormholeId', '_awaitingTooltipElementRendered', function() {
```

Comment:
Is `wormhole` a misnomer now?

Reply:
Depends on how you look at it, I guess. Conceptually, it's still a bit of a wormhole.

Reply:
An alternative would maybe be `portal`?  I think that's what React calls their wormholes.